<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Characterization of the MHC I immunopeptidome of HaCaT cells</title>
  <meta name="description" content="This is a report for the ImmunoSense project of the HaCaT peptidome +/- DNCB">
  <meta name="generator" content="bookdown 0.7 and GitBook 2.6.7">

  <meta property="og:title" content="Characterization of the MHC I immunopeptidome of HaCaT cells" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This is a report for the ImmunoSense project of the HaCaT peptidome +/- DNCB" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Characterization of the MHC I immunopeptidome of HaCaT cells" />
  
  <meta name="twitter:description" content="This is a report for the ImmunoSense project of the HaCaT peptidome +/- DNCB" />
  

<meta name="author" content="Alistair Bailey">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="methods.html">
<link rel="next" href="peptide-binding-motifs.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />










</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Summary</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a></li>
<li class="chapter" data-level="2" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i><b>2</b> Methods</a><ul>
<li class="chapter" data-level="2.1" data-path="methods.html"><a href="methods.html#cell-culture"><i class="fa fa-check"></i><b>2.1</b> Cell culture</a></li>
<li class="chapter" data-level="2.2" data-path="methods.html"><a href="methods.html#hla-typing-of-hacat-cells"><i class="fa fa-check"></i><b>2.2</b> HLA typing of HaCaT cells</a></li>
<li class="chapter" data-level="2.3" data-path="methods.html"><a href="methods.html#immunoaffinity-purification-and-hplc-fractionation"><i class="fa fa-check"></i><b>2.3</b> Immunoaffinity purification and HPLC fractionation</a></li>
<li class="chapter" data-level="2.4" data-path="methods.html"><a href="methods.html#mass-spectrometry"><i class="fa fa-check"></i><b>2.4</b> Mass spectrometry</a></li>
<li class="chapter" data-level="2.5" data-path="methods.html"><a href="methods.html#data-analysis"><i class="fa fa-check"></i><b>2.5</b> Data analysis</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="peptidome-size-and-proportions.html"><a href="peptidome-size-and-proportions.html"><i class="fa fa-check"></i><b>3</b> Peptidome size and proportions</a><ul>
<li class="chapter" data-level="3.1" data-path="peptidome-size-and-proportions.html"><a href="peptidome-size-and-proportions.html#summary-tables"><i class="fa fa-check"></i><b>3.1</b> Summary tables</a></li>
<li class="chapter" data-level="3.2" data-path="peptidome-size-and-proportions.html"><a href="peptidome-size-and-proportions.html#histograms"><i class="fa fa-check"></i><b>3.2</b> Histograms</a></li>
<li class="chapter" data-level="3.3" data-path="peptidome-size-and-proportions.html"><a href="peptidome-size-and-proportions.html#venn-diagrams"><i class="fa fa-check"></i><b>3.3</b> Venn diagrams</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="peptide-binding-motifs.html"><a href="peptide-binding-motifs.html"><i class="fa fa-check"></i><b>4</b> Peptide binding motifs</a><ul>
<li class="chapter" data-level="4.1" data-path="peptide-binding-motifs.html"><a href="peptide-binding-motifs.html#clustering-of-peptides"><i class="fa fa-check"></i><b>4.1</b> Clustering of peptides</a></li>
<li class="chapter" data-level="4.2" data-path="peptide-binding-motifs.html"><a href="peptide-binding-motifs.html#peptide-binding-predictions"><i class="fa fa-check"></i><b>4.2</b> Peptide binding predictions</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="dncb-modifications.html"><a href="dncb-modifications.html"><i class="fa fa-check"></i><b>5</b> DNCB modifications</a><ul>
<li class="chapter" data-level="5.1" data-path="dncb-modifications.html"><a href="dncb-modifications.html#identified-modified-peptides"><i class="fa fa-check"></i><b>5.1</b> Identified modified peptides</a></li>
<li class="chapter" data-level="5.2" data-path="dncb-modifications.html"><a href="dncb-modifications.html#ms-spectra-confirmed-modifications"><i class="fa fa-check"></i><b>5.2</b> MS spectra confirmed modifications</a></li>
<li class="chapter" data-level="5.3" data-path="dncb-modifications.html"><a href="dncb-modifications.html#putative-mhc-peptide-structures"><i class="fa fa-check"></i><b>5.3</b> Putative MHC-peptide structures</a></li>
<li class="chapter" data-level="5.4" data-path="dncb-modifications.html"><a href="dncb-modifications.html#peptide-predictions-for-keratin-type-i-cytoskeletal-13"><i class="fa fa-check"></i><b>5.4</b> Peptide predictions for Keratin type I cytoskeletal 13</a></li>
<li class="chapter" data-level="5.5" data-path="dncb-modifications.html"><a href="dncb-modifications.html#peptide-predictions-for-glutathione-s-transferase-omega-1"><i class="fa fa-check"></i><b>5.5</b> Peptide predictions for Glutathione S-transferase omega-1</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="next-steps.html"><a href="next-steps.html"><i class="fa fa-check"></i><b>6</b> Next steps</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Characterization of the MHC I immunopeptidome of HaCaT cells</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="peptidome-size-and-proportions" class="section level1">
<h1><span class="header-section-number">3</span> Peptidome size and proportions</h1>
<p>Three biological replicates of control and treatment cells were cultured, these were then split in half to create six technical replicates for purification and analysis:</p>
<ul>
<li>1A and 1B</li>
<li>2A and 2B</li>
<li>3A and 3B</li>
</ul>
<p>The raw spectrum files were analysed using <a href="http://www.bioinfor.com/peaks-studio/">Peaks Studio</a> <span class="citation">(J. Zhang et al. <a href="#ref-zhang2012">2012</a>)</span> and then processed further in R <span class="citation">(R Core Team <a href="#ref-R-base">2018</a>)</span>.</p>
<p>Contaminant peptides were removed and I consider peptides only of length 8-15 amino acids in length as real MHC I peptides. Longer peptides may be real peptides, but this conservative approach increases the confidence of my observations.</p>
<div id="summary-tables" class="section level2">
<h2><span class="header-section-number">3.1</span> Summary tables</h2>
<p>Table <a href="peptidome-size-and-proportions.html#tab:summary-tab">3.1</a> shows the number of peptides from each replicate of peptide length 8 to 15 amino acids.</p>
<table>
<caption><span id="tab:summary-tab">Table 3.1: </span>Peptidome 8-15mer totals</caption>
<thead>
<tr class="header">
<th align="left">Replicate</th>
<th align="right">Number of peptides</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Control 1A</td>
<td align="right">1677</td>
</tr>
<tr class="even">
<td align="left">Control 1B</td>
<td align="right">2400</td>
</tr>
<tr class="odd">
<td align="left">Control 2A</td>
<td align="right">2051</td>
</tr>
<tr class="even">
<td align="left">Control 2B</td>
<td align="right">1018</td>
</tr>
<tr class="odd">
<td align="left">Control 3A</td>
<td align="right">5130</td>
</tr>
<tr class="even">
<td align="left">Control 3B</td>
<td align="right">5738</td>
</tr>
<tr class="odd">
<td align="left">DNCB 1A</td>
<td align="right">1427</td>
</tr>
<tr class="even">
<td align="left">DNCB 1B</td>
<td align="right">2042</td>
</tr>
<tr class="odd">
<td align="left">DNCB 2A</td>
<td align="right">1482</td>
</tr>
<tr class="even">
<td align="left">DNCB 2B</td>
<td align="right">1817</td>
</tr>
<tr class="odd">
<td align="left">DNCB 3A</td>
<td align="right">4727</td>
</tr>
<tr class="even">
<td align="left">DNCB 3B</td>
<td align="right">4831</td>
</tr>
</tbody>
</table>
<p>Table <a href="peptidome-size-and-proportions.html#tab:prop-tab">3.2</a> shows the proportions of 9-mer peptides in each replicate. As 9-mer is the predominant peptide length for MHC I molecules, we use the proportion of 9-mers as a benchmark for peptidome purity and replication.</p>
<table>
<caption><span id="tab:prop-tab">Table 3.2: </span>Peptidome 9-mer proportions</caption>
<thead>
<tr class="header">
<th align="left">Replicate</th>
<th align="right">9-mer percent</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Control 1A</td>
<td align="right">53</td>
</tr>
<tr class="even">
<td align="left">Control 1B</td>
<td align="right">52</td>
</tr>
<tr class="odd">
<td align="left">Control 2A</td>
<td align="right">54</td>
</tr>
<tr class="even">
<td align="left">Control 2B</td>
<td align="right">52</td>
</tr>
<tr class="odd">
<td align="left">Control 3A</td>
<td align="right">55</td>
</tr>
<tr class="even">
<td align="left">Control 3B</td>
<td align="right">51</td>
</tr>
<tr class="odd">
<td align="left">DNCB 1A</td>
<td align="right">56</td>
</tr>
<tr class="even">
<td align="left">DNCB 1B</td>
<td align="right">57</td>
</tr>
<tr class="odd">
<td align="left">DNCB 2A</td>
<td align="right">47</td>
</tr>
<tr class="even">
<td align="left">DNCB 2B</td>
<td align="right">53</td>
</tr>
<tr class="odd">
<td align="left">DNCB 3A</td>
<td align="right">55</td>
</tr>
<tr class="even">
<td align="left">DNCB 3B</td>
<td align="right">55</td>
</tr>
</tbody>
</table>
<p>Table <a href="peptidome-size-and-proportions.html#tab:unique-peps">3.3</a> shows the number of unique peptides found across all replicates.</p>
<table>
<caption><span id="tab:unique-peps">Table 3.3: </span>Number of unique peptides</caption>
<thead>
<tr class="header">
<th align="left">Treatment</th>
<th align="right">Number of peptides</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Control</td>
<td align="right">7611</td>
</tr>
<tr class="even">
<td align="left">DNCB</td>
<td align="right">6687</td>
</tr>
</tbody>
</table>
</div>
<div id="histograms" class="section level2">
<h2><span class="header-section-number">3.2</span> Histograms</h2>
<p>Figure <a href="peptidome-size-and-proportions.html#fig:hist-n">3.1</a> shows the distributions of peptide lengths in each peptidome in absolute numbers.</p>


<div class="figure"><span id="fig:hist-n"></span>
<img src="_main_files/figure-html/hist-n-1.png" alt="Histogram of peptide numbers for peptides of 9-15 amino acids. This plot shows a large degree of variation in total numbers of peptides between replicates." width="672" />
<p class="caption">
Figure 3.1: Histogram of peptide numbers for peptides of 9-15 amino acids. This plot shows a large degree of variation in total numbers of peptides between replicates.
</p>
</div>
<p>Figure <a href="peptidome-size-and-proportions.html#fig:hist-p">3.2</a> shows these distributions in terms of proportions of peptide lengths in each peptidome.</p>
<div class="figure"><span id="fig:hist-p"></span>
<img src="_main_files/figure-html/hist-p-1.png" alt="Histogram of proportion of peptide lengths, 9-15 amino acids. This plot shows good replication in terms of relative peptide length frequency between replicates." width="672" />
<p class="caption">
Figure 3.2: Histogram of proportion of peptide lengths, 9-15 amino acids. This plot shows good replication in terms of relative peptide length frequency between replicates.
</p>
</div>
<p>The variation in peptidome size between replicates we believe is primarily due to improvements in the purification and analysis methodology during the generation of this data. Hence the final replicate is yields the largest number of peptides.</p>
</div>
<div id="venn-diagrams" class="section level2">
<h2><span class="header-section-number">3.3</span> Venn diagrams</h2>
<p>To compare the peptidomes of control and treated cells, I first removed all modifications from the peptides such that I compare peptides in their unmodified form. I then selected the unique peptides across all replicates to create combined peptidomes for control and treatment cases.</p>
<p>Figure <a href="peptidome-size-and-proportions.html#fig:venn">3.3</a> indicates that the majority of peptides are the same under treatment and control. Further analysis is required to establish whether there is any significance to the populations not shared between treatment and control, but this data suggests that the core peptidome repertoire is shared between control and DNCB treatment.</p>

<div class="figure" style="text-align: center"><span id="fig:venn"></span>
<img src="_main_files/figure-html/venn-1.png" alt="Venn diagrams of unique Control and DNCB peptides" width="672" />
<p class="caption">
Figure 3.3: Venn diagrams of unique Control and DNCB peptides
</p>
</div>

</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-zhang2012">
<p>Zhang, Jing, Lei Xin, Baozhen Shan, Weiwu Chen, Mingjie Xie, Denis Yuen, Weiming Zhang, Zefeng Zhang, Gilles A Lajoie, and Bin Ma. 2012. “PEAKS Db: De Novo Sequencing Assisted Database Search for Sensitive and Accurate Peptide Identification.” <em>Molecular &amp; Cellular Proteomics</em> 11 (4). ASBMB: M111–010587.</p>
</div>
<div id="ref-R-base">
<p>R Core Team. 2018. <em>R: A Language and Environment for Statistical Computing</em>. Vienna, Austria: R Foundation for Statistical Computing. <a href="https://www.R-project.org/" class="uri">https://www.R-project.org/</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="methods.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="peptide-binding-motifs.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"download": ["_main.pdf", "_main.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
