<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Characterization of the MHC I immunopeptidome of HaCaT cells</title>
  <meta name="description" content="This is a report for the ImmunoSense project of the HaCaT peptidome +/- DNCB">
  <meta name="generator" content="bookdown 0.7 and GitBook 2.6.7">

  <meta property="og:title" content="Characterization of the MHC I immunopeptidome of HaCaT cells" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This is a report for the ImmunoSense project of the HaCaT peptidome +/- DNCB" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Characterization of the MHC I immunopeptidome of HaCaT cells" />
  
  <meta name="twitter:description" content="This is a report for the ImmunoSense project of the HaCaT peptidome +/- DNCB" />
  

<meta name="author" content="Alistair Bailey">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="peptidome-size-and-proportions.html">
<link rel="next" href="dncb-modifications.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />










</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Summary</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a></li>
<li class="chapter" data-level="2" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i><b>2</b> Methods</a><ul>
<li class="chapter" data-level="2.1" data-path="methods.html"><a href="methods.html#cell-culture"><i class="fa fa-check"></i><b>2.1</b> Cell culture</a></li>
<li class="chapter" data-level="2.2" data-path="methods.html"><a href="methods.html#hla-typing-of-hacat-cells"><i class="fa fa-check"></i><b>2.2</b> HLA typing of HaCaT cells</a></li>
<li class="chapter" data-level="2.3" data-path="methods.html"><a href="methods.html#immunoaffinity-purification-and-hplc-fractionation"><i class="fa fa-check"></i><b>2.3</b> Immunoaffinity purification and HPLC fractionation</a></li>
<li class="chapter" data-level="2.4" data-path="methods.html"><a href="methods.html#mass-spectrometry"><i class="fa fa-check"></i><b>2.4</b> Mass spectrometry</a></li>
<li class="chapter" data-level="2.5" data-path="methods.html"><a href="methods.html#data-analysis"><i class="fa fa-check"></i><b>2.5</b> Data analysis</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="peptidome-size-and-proportions.html"><a href="peptidome-size-and-proportions.html"><i class="fa fa-check"></i><b>3</b> Peptidome size and proportions</a><ul>
<li class="chapter" data-level="3.1" data-path="peptidome-size-and-proportions.html"><a href="peptidome-size-and-proportions.html#summary-tables"><i class="fa fa-check"></i><b>3.1</b> Summary tables</a></li>
<li class="chapter" data-level="3.2" data-path="peptidome-size-and-proportions.html"><a href="peptidome-size-and-proportions.html#histograms"><i class="fa fa-check"></i><b>3.2</b> Histograms</a></li>
<li class="chapter" data-level="3.3" data-path="peptidome-size-and-proportions.html"><a href="peptidome-size-and-proportions.html#venn-diagrams"><i class="fa fa-check"></i><b>3.3</b> Venn diagrams</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="peptide-binding-motifs.html"><a href="peptide-binding-motifs.html"><i class="fa fa-check"></i><b>4</b> Peptide binding motifs</a><ul>
<li class="chapter" data-level="4.1" data-path="peptide-binding-motifs.html"><a href="peptide-binding-motifs.html#clustering-of-peptides"><i class="fa fa-check"></i><b>4.1</b> Clustering of peptides</a></li>
<li class="chapter" data-level="4.2" data-path="peptide-binding-motifs.html"><a href="peptide-binding-motifs.html#peptide-binding-predictions"><i class="fa fa-check"></i><b>4.2</b> Peptide binding predictions</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="dncb-modifications.html"><a href="dncb-modifications.html"><i class="fa fa-check"></i><b>5</b> DNCB modifications</a><ul>
<li class="chapter" data-level="5.1" data-path="dncb-modifications.html"><a href="dncb-modifications.html#identified-modified-peptides"><i class="fa fa-check"></i><b>5.1</b> Identified modified peptides</a></li>
<li class="chapter" data-level="5.2" data-path="dncb-modifications.html"><a href="dncb-modifications.html#ms-spectra-confirmed-modifications"><i class="fa fa-check"></i><b>5.2</b> MS spectra confirmed modifications</a></li>
<li class="chapter" data-level="5.3" data-path="dncb-modifications.html"><a href="dncb-modifications.html#putative-mhc-peptide-structures"><i class="fa fa-check"></i><b>5.3</b> Putative MHC-peptide structures</a></li>
<li class="chapter" data-level="5.4" data-path="dncb-modifications.html"><a href="dncb-modifications.html#peptide-predictions-for-keratin-type-i-cytoskeletal-13"><i class="fa fa-check"></i><b>5.4</b> Peptide predictions for Keratin type I cytoskeletal 13</a></li>
<li class="chapter" data-level="5.5" data-path="dncb-modifications.html"><a href="dncb-modifications.html#peptide-predictions-for-glutathione-s-transferase-omega-1"><i class="fa fa-check"></i><b>5.5</b> Peptide predictions for Glutathione S-transferase omega-1</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="next-steps.html"><a href="next-steps.html"><i class="fa fa-check"></i><b>6</b> Next steps</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Characterization of the MHC I immunopeptidome of HaCaT cells</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="peptide-binding-motifs" class="section level1">
<h1><span class="header-section-number">4</span> Peptide binding motifs</h1>
<p>Having quantified the peptidome, to further confirm that these peptides are MHC I peptides I did two analysis:</p>
<ol style="list-style-type: decimal">
<li><p>I used the GibbsCluster tool <span class="citation">(M. Andreatta, Alvarez, and Nielsen <a href="#ref-andreatta2017">2017</a>)</span> to group the 9mer peptides according to their amino acid sequence to identify motifs and compared these groupings to motifs identified for peptides matching the HaCat MHC allotypes in the curated <a href="http://www.iedb.org/">immune epitope database</a> (IEDB) <span class="citation">(Vita et al. <a href="#ref-vita2015">2015</a>)</span>.</p></li>
<li><p>Using <a href="http://www.cbs.dtu.dk/services/NetMHC/">netMHC</a> <span class="citation">(M. Nielsen et al. <a href="#ref-nielsen2003">2003</a>; Massimo Andreatta and Nielsen <a href="#ref-andreatta2016">2016</a>)</span> predictions for binding affinities of clustered 9mer peptides to their identified MHC I allotypes were made to cross validate the clustering.</p></li>
</ol>
<div id="clustering-of-peptides" class="section level2">
<h2><span class="header-section-number">4.1</span> Clustering of peptides</h2>
<p>The unique 9mer peptides for treatment and control with post-translational and/or DNCB modifications were clustered using GibbsCluster <span class="citation">(M. Andreatta, Alvarez, and Nielsen <a href="#ref-andreatta2017">2017</a>)</span>.</p>
<p>Figure <a href="peptide-binding-motifs.html#fig:logos">4.1</a> indicates that three clusters of peptides. This is indicated by the size of the bar scoring the highest Kullbach-Leibler distance, a measure of information entropy. By comparison to motifs of peptides from the IEDB in Figure <a href="peptide-binding-motifs.html#fig:iedb-logos">4.2</a> we can see that these three clusters correspond with motifs for HLA-A*31, HLA-B*40 and HLA-B*51.</p>
<p>The logos corresponding with each cluster measure strength of the preference for each amino acid at each peptide position, as indicated by the size the letters.</p>
<p>Figure <a href="peptide-binding-motifs.html#fig:iedb-logos">4.2</a> also shows that HLA-C*03 and HLA-C*15 have less clear motifs than the A and B allotypes, sharing biochemically similar preferences at potion 9 with HLA-B*40 and HLA-B*51, and also at position 2 with HLA-B*51. This means that the HLA-C peptides are mixed into these clusters.</p>

<div class="figure" style="text-align: center"><span id="fig:logos"></span>
<img src="_main_files/figure-html/logos-1.png" alt="Gibbs clustering of 9mer peptides." width="672" />
<p class="caption">
Figure 4.1: Gibbs clustering of 9mer peptides.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:iedb-logos"></span>
<img src="_main_files/figure-html/iedb-logos-1.png" alt="Immune epitope database peptide motifs." width="672" />
<p class="caption">
Figure 4.2: Immune epitope database peptide motifs.
</p>
</div>
</div>
<div id="peptide-binding-predictions" class="section level2">
<h2><span class="header-section-number">4.2</span> Peptide binding predictions</h2>
<p>Figure <a href="peptide-binding-motifs.html#fig:netmhc-peps">4.3</a> shows the results of using NetMHC to predict whether the peptides from each cluster bound to their identified motif.</p>
<p>As suggested by the peptide logos, for HLA-A*31:01 which has a clear motif 92% of peptides from this cluster were predicted as binders. For the B allotypes this number was reduced, presumably due to the difficulty in separating these peptides from each other and from the C allotypes.</p>
<p>However, what this does confirm is that the peptides we observe are indeed MHC I peptides and not non-specific 9mers.</p>

<div class="figure" style="text-align: center"><span id="fig:netmhc-peps"></span>
<img src="_main_files/figure-html/netmhc-peps-1.png" alt="NetMHC predicted binders" width="672" />
<p class="caption">
Figure 4.3: NetMHC predicted binders
</p>
</div>

</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-andreatta2017">
<p>Andreatta, M., B. Alvarez, and M. Nielsen. 2017. “GibbsCluster: Unsupervised Clustering and Alignment of Peptide Sequences.” Journal Article. <em>Nucleic Acids Res</em>. doi:<a href="https://doi.org/10.1093/nar/gkx248">10.1093/nar/gkx248</a>.</p>
</div>
<div id="ref-vita2015">
<p>Vita, Randi, James A Overton, Jason A Greenbaum, Julia Ponomarenko, Jason D Clark, Jason R Cantrell, Daniel K Wheeler, et al. 2015. “The Immune Epitope Database (Iedb) 3.0.” <em>Nucleic Acids Research</em> 43 (Database issue): D405–D412. doi:<a href="https://doi.org/10.1093/nar/gku938">10.1093/nar/gku938</a>.</p>
</div>
<div id="ref-nielsen2003">
<p>Nielsen, Morten, Claus Lundegaard, Peder Worning, Sanne Lise Lauemøller, Kasper Lamberth, Søren Buus, Søren Brunak, and Ole Lund. 2003. “Reliable Prediction of T-Cell Epitopes Using Neural Networks with Novel Sequence Representations.” <em>Protein Science : A Publication of the Protein Society</em> 12 (5): 1007–17. doi:<a href="https://doi.org/10.1110/ps.0239403">10.1110/ps.0239403</a>.</p>
</div>
<div id="ref-andreatta2016">
<p>Andreatta, Massimo, and Morten Nielsen. 2016. “Gapped Sequence Alignment Using Artificial Neural Networks: Application to the Mhc Class I System.” <em>Bioinformatics (Oxford, England)</em> 32 (4): 511–17. doi:<a href="https://doi.org/10.1093/bioinformatics/btv639">10.1093/bioinformatics/btv639</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="peptidome-size-and-proportions.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="dncb-modifications.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"download": ["_main.pdf", "_main.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
