<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Characterization of the MHC I immunopeptidome of HaCaT cells</title>
  <meta name="description" content="This is a report for the ImmunoSense project of the HaCaT peptidome +/- DNCB">
  <meta name="generator" content="bookdown 0.7 and GitBook 2.6.7">

  <meta property="og:title" content="Characterization of the MHC I immunopeptidome of HaCaT cells" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This is a report for the ImmunoSense project of the HaCaT peptidome +/- DNCB" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Characterization of the MHC I immunopeptidome of HaCaT cells" />
  
  <meta name="twitter:description" content="This is a report for the ImmunoSense project of the HaCaT peptidome +/- DNCB" />
  

<meta name="author" content="Alistair Bailey">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="introduction.html">
<link rel="next" href="peptidome-size-and-proportions.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />










</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Summary</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a></li>
<li class="chapter" data-level="2" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i><b>2</b> Methods</a><ul>
<li class="chapter" data-level="2.1" data-path="methods.html"><a href="methods.html#cell-culture"><i class="fa fa-check"></i><b>2.1</b> Cell culture</a></li>
<li class="chapter" data-level="2.2" data-path="methods.html"><a href="methods.html#hla-typing-of-hacat-cells"><i class="fa fa-check"></i><b>2.2</b> HLA typing of HaCaT cells</a></li>
<li class="chapter" data-level="2.3" data-path="methods.html"><a href="methods.html#immunoaffinity-purification-and-hplc-fractionation"><i class="fa fa-check"></i><b>2.3</b> Immunoaffinity purification and HPLC fractionation</a></li>
<li class="chapter" data-level="2.4" data-path="methods.html"><a href="methods.html#mass-spectrometry"><i class="fa fa-check"></i><b>2.4</b> Mass spectrometry</a></li>
<li class="chapter" data-level="2.5" data-path="methods.html"><a href="methods.html#data-analysis"><i class="fa fa-check"></i><b>2.5</b> Data analysis</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="peptidome-size-and-proportions.html"><a href="peptidome-size-and-proportions.html"><i class="fa fa-check"></i><b>3</b> Peptidome size and proportions</a><ul>
<li class="chapter" data-level="3.1" data-path="peptidome-size-and-proportions.html"><a href="peptidome-size-and-proportions.html#summary-tables"><i class="fa fa-check"></i><b>3.1</b> Summary tables</a></li>
<li class="chapter" data-level="3.2" data-path="peptidome-size-and-proportions.html"><a href="peptidome-size-and-proportions.html#histograms"><i class="fa fa-check"></i><b>3.2</b> Histograms</a></li>
<li class="chapter" data-level="3.3" data-path="peptidome-size-and-proportions.html"><a href="peptidome-size-and-proportions.html#venn-diagrams"><i class="fa fa-check"></i><b>3.3</b> Venn diagrams</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="peptide-binding-motifs.html"><a href="peptide-binding-motifs.html"><i class="fa fa-check"></i><b>4</b> Peptide binding motifs</a><ul>
<li class="chapter" data-level="4.1" data-path="peptide-binding-motifs.html"><a href="peptide-binding-motifs.html#clustering-of-peptides"><i class="fa fa-check"></i><b>4.1</b> Clustering of peptides</a></li>
<li class="chapter" data-level="4.2" data-path="peptide-binding-motifs.html"><a href="peptide-binding-motifs.html#peptide-binding-predictions"><i class="fa fa-check"></i><b>4.2</b> Peptide binding predictions</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="dncb-modifications.html"><a href="dncb-modifications.html"><i class="fa fa-check"></i><b>5</b> DNCB modifications</a><ul>
<li class="chapter" data-level="5.1" data-path="dncb-modifications.html"><a href="dncb-modifications.html#identified-modified-peptides"><i class="fa fa-check"></i><b>5.1</b> Identified modified peptides</a></li>
<li class="chapter" data-level="5.2" data-path="dncb-modifications.html"><a href="dncb-modifications.html#ms-spectra-confirmed-modifications"><i class="fa fa-check"></i><b>5.2</b> MS spectra confirmed modifications</a></li>
<li class="chapter" data-level="5.3" data-path="dncb-modifications.html"><a href="dncb-modifications.html#putative-mhc-peptide-structures"><i class="fa fa-check"></i><b>5.3</b> Putative MHC-peptide structures</a></li>
<li class="chapter" data-level="5.4" data-path="dncb-modifications.html"><a href="dncb-modifications.html#peptide-predictions-for-keratin-type-i-cytoskeletal-13"><i class="fa fa-check"></i><b>5.4</b> Peptide predictions for Keratin type I cytoskeletal 13</a></li>
<li class="chapter" data-level="5.5" data-path="dncb-modifications.html"><a href="dncb-modifications.html#peptide-predictions-for-glutathione-s-transferase-omega-1"><i class="fa fa-check"></i><b>5.5</b> Peptide predictions for Glutathione S-transferase omega-1</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="next-steps.html"><a href="next-steps.html"><i class="fa fa-check"></i><b>6</b> Next steps</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Characterization of the MHC I immunopeptidome of HaCaT cells</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="methods" class="section level1">
<h1><span class="header-section-number">2</span> Methods</h1>
<div id="cell-culture" class="section level2">
<h2><span class="header-section-number">2.1</span> Cell culture</h2>
<p>HaCaT cells <span class="citation">(Boukamp et al. <a href="#ref-boukamp1988">1988</a>)</span> purchased from <a href="http://clsgmbh.de/">Cell lines service</a> were cultured in DMEM medium supplemented with 4.5g/L glucose and 2mM L-glutamine and 10% fetal bovine serum.</p>
<p>For each replicate two populations were grown to 500 x 10<sup>6</sup> cells each, and then the medium changed to DMEM/F-12 without serum. One population was used for the control and the other population treated with a 10 <span class="math inline">\(\mu\)</span>M 2,4-Dinitrochlorobenzene (DNCB) mixture of DNCB and DNCB-D<sub>3</sub> for 24 hours. Cells were harvested with 0.25% Trypsin-EDTA and then washed with DPBS and pelleted prior to freezing and storage at -80<span class="math inline">\(^\circ\)</span>C.</p>
<p>Three biological replicates of control and treatment pellets were cultured, these were then split in half to create six technical replicates:</p>
<ul>
<li>1A and 1B</li>
<li>2A and 2B</li>
<li>3A and 3B</li>
</ul>
</div>
<div id="hla-typing-of-hacat-cells" class="section level2">
<h2><span class="header-section-number">2.2</span> HLA typing of HaCaT cells</h2>
<p>DNA extraction performed with QIAGEN DNAeasy kit under the supervision of Dr.Â Emma Reeves, and typing done by Southampton General Hospital tissue typing unit.</p>
<p>Confirmation of these results was made using genomics analysis using publicly available datasets and <a href="https://ccb.jhu.edu/software/hisat2/index.shtml">HISAT2</a> <span class="citation">(Kim, Langmead, and Salzberg <a href="#ref-kim2015">2015</a>)</span>.</p>
<p>Together this yielded HLA types:</p>
<ul>
<li>Homozygous HLA-A*31:01</li>
<li>HLA-B*40:01, HLA-B*51:01</li>
<li>HLA-C*03, HLA-C*15</li>
</ul>
<p>There is uncertainty of the C alleles beyond two digit identification.</p>
<p>The literature had indicated that HaCaTs carried HLA-A*02 <span class="citation">(Lemaitre et al. <a href="#ref-lemaitre2004">2004</a>)</span>, but our results disconfirm this observation.</p>
</div>
<div id="immunoaffinity-purification-and-hplc-fractionation" class="section level2">
<h2><span class="header-section-number">2.3</span> Immunoaffinity purification and HPLC fractionation</h2>
<p>Cells were lysed in a buffer of 20 mM Tris-HCL (pH 8.0), 150 mM NaCl, 0.5% IGEPAL, 0.25% Sodium deoxycholate, 1 mM EDTA, 0.2 mM Iodoacetamide and protease inhibitors at 4<span class="math inline">\(^\circ\)</span>C on a roller for 30 minutes.</p>
<p>Cell nuclei were removed by centrifugation at 2,000g for 10 minutes. The remaining lysate was then clarified by centrifugation at 15,000g (13,000 rpm) for 1 hour.</p>
<p>The lysate was then added to a tube containing pan-MHC I binding antibody W6/32 crosslinked to Protein A sepahrose beads at a concentration of 2 mg antibody per mL of beads and incubated on a roller at 4<span class="math inline">\(^\circ\)</span>C for 2 hours.</p>
<p>The lysate and beads were then transferred to a column for washing and elution of the MHC I molecules using 5 mL 10% acetic acid, denaturing the MHC I molecules into a heavy chain, <span class="math inline">\(\beta\)</span><sub>2</sub>m and peptide mixture.</p>
<p>The eluate was then dried in a speed-vac evaporator and re-suspended in 500 <span class="math inline">\(\mu\)</span>L of 0.1% TFA/1% acetonitrile prior to injection into the high performance liquid chromatography (HPLC) system.</p>
<p>A Thermo UltiMate 3000 system using a Chromolith HighResolution RP-18 endcapped 100-4.6 HPLC column were used to separate and collect the peptides for mass spectrometry analysis. 0.5 mL peptide fractions were collected over 8 minutes. These fractions were pooled as odd and even fractions and then dried in a speed-vac evaporator.</p>
<p>These fractions were then re-suspended in 20 <span class="math inline">\(\mu\)</span>L of 1% Formic acid and split into four samples, two odd and two even, for mass spectrometry analysis.</p>
</div>
<div id="mass-spectrometry" class="section level2">
<h2><span class="header-section-number">2.4</span> Mass spectrometry</h2>
<p>For each sample a total of 10 <span class="math inline">\(\mu\)</span>L was injected onto a LC system connected to a Thermo Fusion Tribrid mass spectrometer in Orbitrap-Orbitrap mode for maximum sensitivity.</p>
<p>Using 0.1 % formic acid (buffer A) and 0.1% formic acid in acetonitrile (buffer B) each sample was run on gradient of to 30% buffer B over 110 minutes, followed by a five minute gradient to 80% buffer B.</p>
</div>
<div id="data-analysis" class="section level2">
<h2><span class="header-section-number">2.5</span> Data analysis</h2>
<p>The raw spectrum files were analysed using <a href="http://www.bioinfor.com/peaks-studio/">Peaks Studio</a> <span class="citation">(J. Zhang et al. <a href="#ref-zhang2012">2012</a>)</span>.</p>
<p>Variable modifications were set for N-terminal acetylation (42.01 Da) and methionine oxidation (15.99 Da) and for DNCB (166.00 Da). As the lysis buffer contains iodoacetamide cysteine carbamidomethylation (57.02 Da) was also set as a variable modification.</p>
<p>De-novo peptide identification confidence settings were set at 80%, and a false discovery rate (FDR) of 1% was used for the database searches.</p>
<p>The human proteome UP000005640, UniProt release 2017_05 of 42,186 proteins, was used for database searches.</p>
<p>Downstream analysis of the Peaks Studio identifications was performed in R <span class="citation">(R Core Team <a href="#ref-R-base">2018</a>)</span>.</p>

</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-boukamp1988">
<p>Boukamp, P., R. T. Petrussevska, D. Breitkreutz, J. Hornung, A. Markham, and N. E. Fusenig. 1988. âNormal Keratinization in a Spontaneously Immortalized Aneuploid Human Keratinocyte Cell Line.â Journal Article. <em>J Cell Biol</em> 106 (3): 761â71. <a href="http://www.ncbi.nlm.nih.gov/pubmed/2450098" class="uri">http://www.ncbi.nlm.nih.gov/pubmed/2450098</a>.</p>
</div>
<div id="ref-kim2015">
<p>Kim, Daehwan, Ben Langmead, and Steven L Salzberg. 2015. âHISAT: A Fast Spliced Aligner with Low Memory Requirements.â <em>Nature Methods</em> 12 (4). Nature Publishing Group: 357.</p>
</div>
<div id="ref-lemaitre2004">
<p>Lemaitre, G., J. Lamartine, A. Pitaval, P. Vaigot, J. Garin, S. Bouet, C. Petat, et al. 2004. âExpression Profiling of Genes and Proteins in Hacat Keratinocytes: Proliferating Versus Differentiated State.â Journal Article. <em>J Cell Biochem</em> 93 (5): 1048â62. doi:<a href="https://doi.org/10.1002/jcb.20212">10.1002/jcb.20212</a>.</p>
</div>
<div id="ref-zhang2012">
<p>Zhang, Jing, Lei Xin, Baozhen Shan, Weiwu Chen, Mingjie Xie, Denis Yuen, Weiming Zhang, Zefeng Zhang, Gilles A Lajoie, and Bin Ma. 2012. âPEAKS Db: De Novo Sequencing Assisted Database Search for Sensitive and Accurate Peptide Identification.â <em>Molecular &amp; Cellular Proteomics</em> 11 (4). ASBMB: M111â010587.</p>
</div>
<div id="ref-R-base">
<p>R Core Team. 2018. <em>R: A Language and Environment for Statistical Computing</em>. Vienna, Austria: R Foundation for Statistical Computing. <a href="https://www.R-project.org/" class="uri">https://www.R-project.org/</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="introduction.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="peptidome-size-and-proportions.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"download": ["_main.pdf", "_main.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
